main.py

```diff
@@ -1,13 +1,89 @@
+import csv
 import logging
-import random
 import os
-import csv
 
 import hydra
+import wandb
 from omegaconf import DictConfig, OmegaConf
 
-import wandb
-from ufm import countermeasures, fine_tuning, metrics, models, pretrain_score, unadapt, utils
+from ufm import countermeasures, fine_tuning, metrics, models, unadapt, utils
+
+
+def run_baseline_suite():
+    """
+    Run the suite of methods and metrics for a baseline model
+    """
+    logger = logging.getLogger()
+    logger.info("Running Baseline suite")
+    model_base = models.HuggingFaceModel(wandb.config.model)
+
+    logger.info("Benchmarking base model for relative pre-training performance")
+    metrics.run_benchmark(model_base, "base")
+    ft_val_losses = fine_tuning.run_fine_tune(model_base, wandb.config.finetune)
+
+    # Create a csv file to store the val losses
+    file_name = utils.get_base_finetune_eval_loss_path(wandb.config.baseline_metrics_path, wandb.config.model)
+    logger.info(f"Saving base model val losses to disk: {file_name}")
+    os.makedirs(os.path.dirname(file_name), exist_ok=True)
+    with open(file_name, "w") as f:
+        writer = csv.writer(f)
+        writer.writerow(ft_val_losses)
+
+    logger.info("Benchmarking finetuned model")
+    metrics.run_benchmark(model_base, "finetuned")
+
+    logger.info("Calculating final unadaptability metrics")
+    loss_gap_ratio = metrics.calculate_unadaptability_metrics(
+        ft_val_losses, wandb.config.baseline_metrics_path, wandb.config.model
+    )
+
+    logger.info("This is a baseline run, loss gap ratio should be zero.")
+    logger.info(f"Final loss gap ratio: {loss_gap_ratio:.6f}")
+    logger.info("Finished Baseline suite")
```

Comment:
Maybe you could log an error if it's not close to 0? Not too important though if you'd rather not